libs/blocks/nav-marquee/nav-marquee.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/*
+ * Navigation Marquee
+ */
+import { createTag } from '../../utils/utils.js';
+import { decorateButtons, getBlockSize } from '../../utils/decorate.js';
+
+const DOWN_ARROW_ICON = '<svg aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="10.875" height="13.323" viewBox="0 0 10.875 13.323"><title>down arrow</title><g fill="none" stroke="#1473e6" stroke-linecap="round" stroke-width="2"><path d="M5.478 1v10.909"/><g><path d="m9.461 7.885-4.023 4.023m0 0L1.414 7.885"/></g></g></svg>';
+
+function findAnchorTarget(text) {
+  let linkText = text.toLowerCase();
+  if (linkText.charAt(0) === '_') {
+    linkText = linkText.charAt(0) === '_' ? linkText.substring(1) : linkText;
+    linkText = linkText.replaceAll('_', '-');
+    linkText = linkText.replaceAll(/[ /|&;$%@"<>()+,.]/g, '');
+    linkText = `#${linkText}`;
+  }
+  return linkText;
```

Comment:
I think we should bake in`scroll-margin-top` to the linking or the `<h>` tags. Currently when I click on any links to jump to that section the title for that section gets cut off by the global nav. 

[This will help sort that out](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-margin-top).